README.md

```diff
@@ -91,6 +91,7 @@ Repository | Description
 [Security Cheatsheets](https://github.com/andrewjkerr/security-cheatsheets) 		| Collection of cheatsheets for various infosec tools and topics
 [Security List](https://github.com/zbetcheckin/Security_list)						 | Great security list for fun and profit
 [Shell](https://github.com/alebcay/awesome-shell) 									| List of awesome command-line frameworks, toolkits, guides and gizmos to make complete use of shell
+[The Cyberclopaedia](https://cr0mll.github.io/cyberclopaedia) | The open-source encyclopaedia of cybersecurity
```

Comment:
I believe this section needs to be expanded on.

Reply:
ok